common/app/common/configuration.scala

```diff
@@ -590,7 +590,12 @@ class GuardianConfiguration extends Logging {
 
   object formstack {
     lazy val url = configuration.getMandatoryStringProperty("formstack.url")
-    lazy val oAuthToken = configuration.getMandatoryStringProperty("formstack.oauthToken")
+    object identity {
+      lazy val oauthToken = configuration.getMandatoryStringProperty("formstack.identity.oauthToken")
```

Comment:
There is no need to use `lazy` here. An `object`'s `val` won't be initialised until the object is referenced.